py/tests/scale.py

```diff
@@ -4,49 +4,74 @@
 import time
 import random
 
+def plane_commit(card, crtc, plane, fb, x, y, w, h) :
+	req = pykms.AtomicReq(card)
+	req.add(plane, {"FB_ID": fb.id,
+		"CRTC_ID": crtc.id,
+		"SRC_X": 0 << 16,
+		"SRC_Y": 0 << 16,
+		"SRC_W": fb.width << 16,
+		"SRC_H": fb.height << 16,
+		"CRTC_X": x,
+		"CRTC_Y": y,
+		"CRTC_W": w,
+		"CRTC_H": h,
+		"zpos": 0})
+	r = req.commit_sync()
+
 card = pykms.Card()
 res = pykms.ResourceManager(card)
-conn = res.reserve_connector("hdmi")
+#conn = res.reserve_connector("hdmi")
+conn = card.get_first_connected_connector
 crtc = res.reserve_crtc(conn)
-plane = res.reserve_overlay_plane(crtc)
+#plane = res.reserve_overlay_plane(crtc)
+plane = card.planes[0]
```

Comment:
Why? Plane 0 is GFX plane on DSS2-5, without scaling.

I think we need a helper function somewhere. It could do a test commit with scaling. If it fails, the plane doesn't support scaling. Then again, how reliable is that, as it could fail to numerous reasons...

But perhaps it's easiest to hack this as a cmdline parameter, so that the user can give the plane idx.